test/browser/html5_special_canvas_event_targets.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+#include <assert.h>
+
+#include <emscripten/emscripten.h>
+#include <emscripten/html5.h>
+
+const char *TARGETS[]={"!foovas", "#canvas", "canvas"};
+const char *target = TARGETS[WHICH_TARGET];
```

Comment:
Why not just do `-DTARGET="!foovas"` on the command line rather than use this index/indirection?

Reply:
Good point, I'll delete it.  I was worried about escaping but since there are no spaces it's not so bad.